pynsist.cfg

```diff
@@ -42,18 +42,70 @@ pypi_wheels = PyQt5==5.14.2
     pywin32==227
     send2trash==1.7.1
 files = share/
-exclude = pkgs/PyQt5/QtBluetooth.pyd
-    pkgs/PyQt5/QtDesigner.pyd
-    pkgs/PyQt5/QtQuick.pyd
-    pkgs/PyQt5/QtQuickWidgets.pyd
-    pkgs/PyQt5/QtQml.pyd
-    pkgs/PyQt5/Qt/bin/Qt5Bluetooth.dll
+exclude = pkgs/PyQt5/Qt/bin/Qt5Bluetooth.dll
+    pkgs/PyQt5/Qt/bin/Qt5DBus.dll
     pkgs/PyQt5/Qt/bin/Qt5Designer.dll
+    pkgs/PyQt5/Qt/bin/Qt5Help.dll
+    pkgs/PyQt5/Qt/bin/Qt5Location.dll
+    pkgs/PyQt5/Qt/bin/Qt5Multimedia.dll
+    pkgs/PyQt5/Qt/bin/Qt5MultimediaWidgets.dll
+    pkgs/PyQt5/Qt/bin/Qt5NetworkAuth.dll
+    pkgs/PyQt5/Qt/bin/Qt5Nfc.dll
+    pkgs/PyQt5/Qt/bin/Qt5OpenGL.dll
```

Comment:
Thanks for doing this! Please keep Qt5OpenGL in case we end up using bits of it in our QGraphicsScene rendering layer.